src/lxc/confile.c

```diff
@@ -255,12 +255,21 @@ static const size_t config_jump_table_size = sizeof(config_jump_table) / sizeof(
 struct lxc_config_t *lxc_get_config(const char *key)
 {
 	size_t i;
+	struct lxc_config_t *best_match = NULL;
+	size_t best_match_name_len = 0;
 
-	for (i = 0; i < config_jump_table_size; i++)
-		if (!strncmp(config_jump_table[i].name, key, strlen(config_jump_table[i].name)))
-			return &config_jump_table[i];
+	for (i = 0; i < config_jump_table_size; i++) {
```

Comment:
Me, that means we always run through the whole table, no? That sounds suboptimal.
Wouldn't it just suffice if you placed the `lxc.selinux.context.keyring` key at a different location? This already works for other keys, no?

Reply:
Yes, that would be sufficient. If this is the preferred way, I can simply do this. But I guess somebody will run into that issue again...

Reply:
Hm, I thought we had documented that behavior... Weird. If you want, you can add a separate commit here to this branch that adds a comment above the jump table explaining this behavior.

Reply:
hmm, at least i didn't stumble across that... Nevertheless, I added a comment ;-)

Reply:
Thanks! :)